scripts/globals/mobskills/PW_Gorgon_Dance.lua

```diff
@@ -7,25 +7,26 @@
 -- Range: 10' radial?
 -- Notes: Used only by Medusa. Starts using it at 25%.
 ---------------------------------------------
+
 require("scripts/globals/settings");
 require("scripts/globals/status");
 require("scripts/globals/monstertpmoves");
+
 ---------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
-    local mobSkin = mob:getModelId();
-
-    if (mobSkin == 1865) then
+    
+    local mobhp = mob:getHPP();
+    if (mobhp <= 25) then -- She's under 25%, it's okay to use this.

```

Comment:
Like there's no guarantee that it be in Medusa form below 25%, might be above that and then change to something else before it reaches 25%.